swift/schema.py

```diff
@@ -1239,3 +1239,10 @@ class ParameterizedProtocolType(Type):
 
 class AbiSafeConversionExpr(ImplicitConversionExpr):
     pass
+
+
+class SingleValueStmtExpr(Expr):
+    """
+    An expression that may wrap a statement which produces a single value.
```

Comment:
"may" makes it sound like the statement would be optional. Maybe
```suggestion
    An expression that wraps a statement which produces a single value.
```

Reply:
That's what the docs from compiler sources said 🤷

Reply:
Hmm, then we should probably double-check whether `Stmt` should not be `optional` (I would be surprised if it could though). If not, then I would drop the `may` regardless of the internal compiler docs (which by the way we shouldn't really copy :wink:).